mvx-esdt-safe/tests/mvx_esdt_safe_blackbox_tests.rs

```diff
@@ -3068,3 +3069,157 @@ fn test_update_config() {
+                SOV_ID_1.to_token_identifier(),
+            ));
+            sc.multiversx_to_sovereign_token_id_mapper(&EgldOrEsdtTokenIdentifier::esdt(
+                SECOND_TOKEN_ID,
+            ))
+            .set(EgldOrEsdtTokenIdentifier::esdt(
+                SOV_ID_2.to_token_identifier(),
+            ));
+            sc.sovereign_to_multiversx_token_id_mapper(&EgldOrEsdtTokenIdentifier::esdt(SOV_ID_1))
+                .set(EgldOrEsdtTokenIdentifier::esdt(
+                    FIRST_TOKEN_ID.to_token_identifier(),
+                ));
+            sc.sovereign_to_multiversx_token_id_mapper(&EgldOrEsdtTokenIdentifier::esdt(SOV_ID_2))
+                .set(EgldOrEsdtTokenIdentifier::esdt(
+                    SECOND_TOKEN_ID.to_token_identifier(),
+                ));
+        });
+
+    state
+        .common_setup
+        .deploy_chain_config(OptionalValue::None, None);
+
+    let token_data = EsdtTokenData {
+        amount: BigUint::from(ONE_HUNDRED_THOUSAND),
+        ..Default::default()
+    };
+
+    let first_payment = OperationEsdtPayment::new(
+        EgldOrEsdtTokenIdentifier::esdt(SOV_ID_1),
+        0,
+        token_data.clone(),
+    );
+
+    let second_payment = OperationEsdtPayment::new(
+        EgldOrEsdtTokenIdentifier::esdt(TRUSTED_TOKEN_IDS[0]),
+        0,
+        token_data.clone(),
+    );
+
+    let third_payment =
+        OperationEsdtPayment::new(EgldOrEsdtTokenIdentifier::esdt(SOV_ID_2), 0, token_data);
+
+    let operation = Operation::new(
+        TESTING_SC_ADDRESS.to_managed_address(),
+        vec![first_payment, second_payment, third_payment].into(),
+        OperationData::new(1, USER_ADDRESS.to_managed_address(), None),
+    );
+    let operation_hash = state.common_setup.get_operation_hash(&operation);
+    let hash_of_hashes = ManagedBuffer::new_from_bytes(&sha256(&operation_hash.to_vec()));
+
+    let (signature, public_keys) = state.common_setup.get_sig_and_pub_keys(1, &hash_of_hashes);
+
+    state.common_setup.register(
+        public_keys.first().unwrap(),
+        &MultiEgldOrEsdtPayment::new(),
+        None,
+    );
+
+    state.common_setup.complete_chain_config_setup_phase();
+
+    state
+        .common_setup
+        .deploy_header_verifier(vec![ScArray::ChainConfig, ScArray::ESDTSafe]);
+
+    state
+        .common_setup
+        .complete_header_verifier_setup_phase(None);
+
+    state.set_token_burn_mechanism(TRUSTED_TOKEN_IDS[0], None);
+
+    let bitmap = ManagedBuffer::new_from_bytes(&[0x01]);
+    let epoch = 0;
+
+    state.common_setup.register_operation(
+        USER_ADDRESS,
+        signature,
+        &hash_of_hashes,
+        bitmap,
+        epoch,
+        MultiValueEncoded::from_iter(vec![operation_hash]),
+    );
+
+    state.execute_operation(
+        &hash_of_hashes,
+        &operation,
+        Some(vec![
+            DEPOSIT_EVENT,
+            &SOV_ID_1.as_str(),
+            &TRUSTED_TOKEN_IDS[0],
+            &SOV_ID_2.as_str(),
```

Comment:
Now that we can check an array of logs, let's also add `EXECUTED_BRIDGE_OP_EVENT`